heron_wsgi/admin_lib/heron_policy.py

```diff
@@ -871,10 +871,10 @@ def _script():
         cwd = Path('.', open=io_open, joinpath=joinpath, listdir=listdir)
         logging.basicConfig(level=logging.DEBUG, stream=stderr)
 
-        sys_path.append('..')
+        sys_path.append('./heron_wsgi/')
```

Comment:
This would be quite inconvenient. It's a long-standing convention that `cd admin_lib; python heron_policy.py WHO` is the way to run this integration test.

Can you `cd admin_lib` within the `docker exec` world?

Reply:
well i think some of this has to do with all the duplicate config files that used to exist.

Reply:
They weren't duplicates; they were just split up in a way that's convenient for integration testing rather than production use.